nautobot_ssot/jobs/base.py

```diff
@@ -110,6 +110,18 @@ def sync_data(self):
         - self.sync       (Sync instance tracking this job execution)
```

Comment:
New screenshot (above) is quite large - at the very least, consider narrowing your browser window to the minimum size that still shows all of the data usefully, as there's a lot of horizontal whitespace that could be omitted.